railties/test/application/test_runner_test.rb

```diff
@@ -446,7 +446,7 @@ def test_pass_TEST_env_on_rake_test
       create_test_file :models, 'account'
       create_test_file :models, 'post', pass: false
 
-      output =  Dir.chdir(app_path) { `bin/rake test TEST=test/models/post_test.rb` }
+      output =  Dir.chdir(app_path) { `bin/rails test TEST=test/models/post_test.rb` }
```

Comment:
Original fix was https://github.com/vipulnsward/rails/commit/64448c29deb2f7239ac7e21035c05b8793ee1f47 , works for rake only.